docs/cis_self_assessment.md

```diff
@@ -1415,51 +1373,39 @@ Note: This recommendation only applies if you let kubelets get their certificate
 
 **Result:** Pass
```

Comment:
We do not pass this one. Need to discuss.

Reply:
Updated.